packages/api/package.json

```diff
@@ -59,6 +59,7 @@
     "graphql-tools": "^5.0.0",
     "joi": "^17.6.0",
     "jsonwebtoken": "^8.5.1",
+    "luxon": "^3.0.3",
```

Comment:
For me date-fns is just bulky and not as easy to use as luxon and i have made a lot of test with the code with luxon so if possible i would prefer to keep it this way because otherwise i will need to invest another half/full day for testing again.